Docs/Queuing.md

```diff
@@ -14,6 +14,18 @@ services.AddQueue();
 
 That's it! This will automatically register the queue in your service container.
 
+## Configuration
```

Comment:
I would move this to just after the "Global Error Handling" section. The idea is that it's less friction for someone coming into the docs to just find out how to queue the tasks. I think error handling would be more important to a user up-front.

The more "stuff" we put before the "meat" of the queue docs, the more a user has to search through the docs to just figure out the basic way to get it to work. So I've tried to order the docs in this way so that the first few sections are the essential parts needed, and supplemental parts come after. Just to make it clear what my philosophy is here so this change doesn't seem arbitrary ;)

I would even title this section `Specifying The Consummation Delay` just to be more explicit and obvious so the reader can quickly see what features are available to them at a quick glance.

Reply:
That makes sense :) I will get on it.